cloudify_agent/installer/__init__.py

```diff
@@ -49,7 +49,7 @@ def run_agent_command(self, command, execution_env=None):
         if execution_env is None:
             execution_env = {}
 
-        # TBD: Investigate why adding quotes don't work on linux
+        # TBD: Investigate why adding quotes doesn't work on linux
```

Comment:
Thanks for your comment. Looking at the code, maybe the problem could be fixed by removing the quotes in `AgentInstaller` and moving them to `Windows/LinuxInstallerMixing.cfy_agent_path`. This way, it should be possible to quote both `python` and `cfy-agent`. I'll give it a try in a separate PR.